Docker build failes

Hey!

seems docker build doesnt work.
Get the following error:

`Step 5 : RUN go build -o /go/bin/slow_cooker /go/src/slow_cooker/main.go
 ---> Running in ffcdca4af5a4
main.go:25:2: cannot find package "github.com/buoyantio/slow_cooker/hdrreport" in any of:
	/go/src/slow_cooker/vendor/github.com/buoyantio/slow_cooker/hdrreport (vendor tree)
	/usr/local/go/src/github.com/buoyantio/slow_cooker/hdrreport (from $GOROOT)
	/go/src/github.com/buoyantio/slow_cooker/hdrreport (from $GOPATH)
main.go:26:2: cannot find package "github.com/buoyantio/slow_cooker/ring" in any of:
	/go/src/slow_cooker/vendor/github.com/buoyantio/slow_cooker/ring (vendor tree)
	/usr/local/go/src/github.com/buoyantio/slow_cooker/ring (from $GOROOT)
	/go/src/github.com/buoyantio/slow_cooker/ring (from $GOPATH)
main.go:27:2: cannot find package "github.com/buoyantio/slow_cooker/window" in any of:
	/go/src/slow_cooker/vendor/github.com/buoyantio/slow_cooker/window (vendor tree)
	/usr/local/go/src/github.com/buoyantio/slow_cooker/window (from $GOROOT)
	/go/src/github.com/buoyantio/slow_cooker/window (from $GOPATH)
The command '/bin/sh -c go build -o /go/bin/slow_cooker /go/src/slow_cooker/main.go' returned a non-zero code: 1`

My setup:
`docker version
Client:
 Version:      1.12.3
 API version:  1.24
 Go version:   go1.7.3
 Git commit:   6b644ec
 Built:        Thu Oct 27 19:42:59 2016
 OS/Arch:      linux/amd64

Server:
 Version:      1.12.3
 API version:  1.24
 Go version:   go1.7.3
 Git commit:   6b644ec
 Built:        Thu Oct 27 19:42:59 2016
 OS/Arch:      linux/amd64`

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,10 +1,9 @@
-FROM library/golang:1.7.1
+FROM library/golang:1.7.3
 
 WORKDIR /go/src/slow_cooker
 
-ADD ./main.go /go/src/slow_cooker/
-ADD ./vendor /go/src/slow_cooker/vendor
+ADD . /go/src/github.com/buoyantio/slow_cooker
 
-RUN go build -o /go/bin/slow_cooker /go/src/slow_cooker/main.go
+RUN go build -o /go/bin/slow_cooker /go/src/github.com/buoyantio/slow_cooker/main.go
 
 ENTRYPOINT ["/go/bin/slow_cooker"]

